tests/terraform_compliance/steps/its_key_value_test_cases/k_case.json

```diff
@@ -0,0 +1,10 @@
+{
+    "stash": [
+        {
+            "id": "a",
+            "FoO": 1000
+        }
+    ],
+    "args": ["fOo", 1000],
+    "expected_ids": []
```

Comment:
Would we expect key comparisons to be case insensitive?

Reply:
Yes.

Reply:
Okay. I'll update this test and try and fix the code.